.env

```diff
@@ -1,4 +1,9 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env`
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
\ No newline at end of file
+## NB only variables prefixed with SIMORGH_ are available on both the server and client
+SIMORGH_BASE_URL=http://localhost:7080
```

Comment:
We're supposed to avoid committing the `.env` in right? I've been confused as to why it's not on .gitignore for while. I guess because we want the base file to exist?

Reply:
Oops. Yeah, that's the case.